integration_test/install_test.go

```diff
@@ -105,6 +110,75 @@ func TestKrewInstall_Manifest(t *testing.T) {
 	test.AssertExecutableInPATH("kubectl-" + validPlugin)
 }
 
+func startLocalServer() (*httptest.Server, error) {
+	server := httptest.NewUnstartedServer(http.FileServer(http.Dir("testdata")))
+
+	listener, err := net.Listen("tcp", localServerURL)
+	if err != nil {
+		return nil, errors.New("trouble starting local server")
+	}
+	server.Listener = listener
+	server.Start()
+	return server, nil
+}
+
+func TestKrewInstall_ManifestURL(t *testing.T) {
+	skipShort(t)
+
+	server, err := startLocalServer()
+	if err != nil {
```

Comment:
Do this:

Pass t to startLocalServer, Mark that method as t.Helper() and if err starting server, t.Fatal in that method; not here.

Reply:
Done